components/progressbar/metadata/progressbar.yml

```diff
@@ -1,128 +1,152 @@
 name: Progress bar
 SpectrumSiteSlug: https://spectrum.adobe.com/page/progress-bar/
+status: Verified
+id: progressbar-m
 sections:
   - name: Migration Guide
     description: |
-        ### Component renamed
-        Bar loader is now known as Progress bar. Replace all `.spectrum-BarLoader*` classnames with `.spectrum-ProgressBar*`.
+      ### Component renamed
+      Bar loader is now known as Progress bar. Replace all `.spectrum-BarLoader*` classnames with `.spectrum-ProgressBar*`.
+
+      ### T-shirt sizing
+      Progress bar now supports t-shirt sizing and requires that you specify the size of button by adding a `.spectrum-ProgressBar--size*` class.
+
+      ### Size classnames changed
+      Previous size classnames map as follows:
+
+      | Previous size classname         | New size classname                |
+      | ------------------------------- | --------------------------------- |
+      | `.spectrum-ProgressBar--small`  | `.spectrum-ProgressBar--sizeS`    |
+      | `.spectrum-ProgressBar--large`  | `.spectrum-ProgressBar--sizeM`    |
```

Comment:
Yeah, the table is meant to emphasize that the classnames changed. I suppose we could write some additional verbiage here, but the migration guide is supposed to be this and I think the big table is enough. If you want to suggest some verbiage we can review it.

Note that I didn't reference `L` and `XL` here because there was no corresponding previous variant, this is a migration guide to show you how to get from old to new, not an API table.